laiaaa0/assignment4/Map.h

```diff
@@ -0,0 +1,35 @@
+#include <vector>
+#include <iostream>
+
+#ifndef MAP_H
```

Comment:
Minor thing: this usually goes before everything else. It doesn't really mater because standard headers have their own guards and will not be included twice tough.